app/gather/assets/apps/surveyor/index.jsx

```diff
@@ -40,6 +41,10 @@ class SurveyorDispatcher extends Component {
           {
             name: 'surveyor',
             url: getSurveyorsAPIPath({ id: surveyorId })
+          },
+          {
+            name: 'surveys',
+            url: getSurveysAPIPath({ app: ODK_APP, fields: ['project_id', 'name'].join(',') })
```

Comment:
You need to add this also for the "add" action :grimacing:

Reply:
👍